go.mod

```diff
@@ -105,8 +105,7 @@ require (
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/protobuf v1.5.4 // indirect
-	github.com/google/gofuzz v1.2.0 // indirect
-	github.com/gorilla/websocket v1.5.0 // indirect
+	github.com/gorilla/websocket v1.5.4-0.20250319132907-e064f32e3674 // indirect
```

Comment:
Do we know why this is pointing to an unreleased version? Was this a required update (and do we need to ask the maintainers for a new release?) or was this over-enthusiastically updated by some of our dependencies?

Reply:
It was introduced by `k8s.io/client-go v0.33.0`.

https://github.com/kubernetes/client-go/blob/bf3d45b702d837ffb83203a410e8f0e3b2b68637/go.mod#L15

Technically we only need cri-api bump to support those new CRI features. So I changed to only bump up cri-api. Hopefully in a later version of `k8s.io/client-go` they will use a released `gorilla/websocket` version :)

Reply:
Hmm.. right; I see there was some discussion here for a v1.5.4 release but one PR pending before that; https://github.com/gorilla/websocket/pull/978#issuecomment-2738632253


@AlexVulaj @liggitt do you know if a v1.5.4 (or v1.6.0 as there's apparently new features (following SemVer)) is in the charts?

Reply:
> Was this a required update

It was required to get https proxy support which we added upstream to the gorilla library in https://github.com/gorilla/websocket/pull/978 in https://github.com/kubernetes/kubernetes/pull/129872 and unblock https://github.com/kubernetes/enhancements/issues/4006

> @AlexVulaj @liggitt do you know if a v1.5.4 (or v1.6.0 as there's apparently new features (following SemVer)) is in the charts?

https://github.com/gorilla/websocket/pull/978#issuecomment-2738632253 is the pending request for a release (either 1.5.x or 1.6.x)